src/main/resources/xl-rules.xml

```diff
@@ -12,124 +12,28 @@
         </conditions>
-        <steps>
-            <os-script>
-                <description expression="true">"Upload project for datical [%s]" % deployed.name</description>
-                <script>datical/datical_upload</script>
-                <order>60</order>
-                <classpath-resources>
-                    <value>7zip/7z1604-x64.exe</value>
-                </classpath-resources>
-            </os-script>
-
-            <os-script>
-                <description expression="true">"Deploy datical project [%s]" % deployed.name</description>
-                <script>datical/datical_deploy</script>
-                <order>63</order>
-                <upload-artifacts>false</upload-artifacts>
-            </os-script>
-
-            <os-script>
-                <description expression="true">"Remove project [%s] from server" % deployed.name</description>
-                <script>datical/datical_remove</script>
-                <order>64</order>
-                <upload-artifacts>false</upload-artifacts>
-            </os-script>
-        </steps>
-    </rule>
+		<planning-script-path>planning/create_plan.py</planning-script-path>
+	</rule>	
 
-    <rule name="datical.DeployedProject.StatusDetail.CREATE" scope="deployed">
+	<rule name="datical.DeployedProject.MODIFY" scope="deployed">
         <conditions>
-            <type>datical.DeployedProject</type>
-            <operation>CREATE</operation>
-            <operation>MODIFY</operation>
-            <expression>deployed.runStatus</expression>
-        </conditions>
-        <steps>
-            <os-script>
-                <description expression="true">"Statusdetail for datical project [%s]" % deployed.name</description>
-                <script>datical/datical_status_detail</script>
-                <order>61</order>
-                <upload-artifacts>false</upload-artifacts>
-            </os-script>
-        </steps>
-    </rule>
-
-    <rule name="datical.DeployedProject.Forecast.CREATE" scope="deployed">
-        <conditions>
-            <type>datical.DeployedProject</type>
-            <operation>CREATE</operation>
+			<type>datical.DeployedProject</type>
             <operation>MODIFY</operation>
-            <expression>deployed.runForecast</expression>
         </conditions>
-        <steps>
-            <os-script>
-                <description expression="true">"Forecast for datical project [%s]" % deployed.name</description>
-                <script>datical/datical_forecast</script>
-                <order>62</order>
-                <upload-artifacts>false</upload-artifacts>
-            </os-script>
-        </steps>
-    </rule>
-
-
-    <rule name="datical.DeployedProject.DESTROY" scope="deployed">
+		<planning-script-path>planning/modify_plan.py</planning-script-path>
+	</rule>	
+	
+	<rule name="datical.DeployedProject.DESTROY" scope="deployed">
         <conditions>
-            <type>datical.DeployedProject</type>
+			<type>datical.DeployedProject</type>
```

Comment:
Indent?

Reply:
Not sure why identation shows up. I'll fix it.